lib/mobius.ex

```diff
@@ -90,8 +92,17 @@ defmodule Mobius do
   * `:name` - the name of the Mobius instance you are using. Unless you
     specified this in your configuration you should be safe to allow this
     option to default, which is `:mobius_metrics`.
+  * `:from` - the unix timestamp, in seconds, for the first record up to the
+    `:to` offset. If no `:to` offset is provided it will default to the current
+     time of the query. If no `:from` option is provided it will default to the
+     last 3 minutes of metric history.
+  * `:to` - the unix timestamp, in seconds, for the last record in the query. If
+    this option is not provided it will default to the current time of the query.
   """
-  @type plot_opt() :: {:name, Mobius.name()}
+  @type plot_opt() ::
+          {:name, Mobius.name()}
+          | {:from, {integer(), time_unit()}}
```

Comment:
```suggestion
          | {:from, integer()}}
```

The docs right above this say that the timestamp is in seconds rather than being a tuple. I don't think that I've ever seen unix time in minute, hour, or day units. I assume that's minutes, hours, or days since Jan 1, 1970. I'm leaning towards the simpler unix-time in seconds only for these arguments to save some type, but let me know if I'm missing the intent.